components/TestComponent.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="bg-rad-fill-ghost text-rad-foreground-contrast">
+    Testing tailwind, rad themed colors and dark-mode support, icons, nuxt-ui (delete me)
```

Comment:
Are these test components (`TestComponent.vue`, `TestDropdown.vue`) meant to be deleted in a future PR?

Reply:
Yup, I put those two in to verify everything works as expected and for the team to have a couple of examples. Feel free to delete them whenever.